addons/event_system_plugin/events/return.gd

```diff
@@ -0,0 +1,12 @@
+extends Event
+
+func _init() -> void:
+	event_color = Color("EB5E55")
+	event_name = "Return"
```

Comment:
```suggestion
	event_name = "End Timeline"
```
Should also change the filename to "end_timeline".

'Return' implies there's something to return _to_, which is false with current implementation.